package-res/resources/web/prompting/PromptPanel.js

```diff
@@ -446,6 +446,7 @@ define(['cdf/lib/Base', 'cdf/Logger', 'dojo/number', 'dojo/i18n', 'common-ui/uti
         onAfterRender: null,
         onBeforeUpdate: null,
         onAfterUpdate: null,
+        onStateChanged: null,
```

Comment:
Need add property description (js doc) for 'onStateChanged' as for other class properties such as 'onBeforeRender'.